packages/react/src/components/RadioTile/RadioTile.tsx

```diff
@@ -5,23 +5,86 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import React, {
+  forwardRef,
+  ChangeEvent,
+  KeyboardEvent,
+  ReactNode,
+  Ref,
+} from 'react';
 import { CheckmarkFilled } from '@carbon/icons-react';
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-import React from 'react';
 import { keys, matches } from '../../internal/keyboard';
 import { useFallbackId } from '../../internal/useId';
 import { usePrefix } from '../../internal/usePrefix';
 import deprecate from '../../prop-types/deprecate';
 import { noopFn } from '../../internal/noopFn';
 import { Text } from '../Text';
 
-const RadioTile = React.forwardRef(function RadioTile(
+interface RadioTileProps {
+  /**
+   * `true` if this tile should be selected.
+   */
+  checked: boolean;
+
+  /**
+   * The tile content.
+   */
+  children: ReactNode;
+
+  /**
+   * The CSS class names.
+   */
+  className?: string;
+
+  /**
+   * Specify whether the RadioTile should be disabled
+   */
+  disabled: boolean;
+
+  /**
+   * The ID of the `<input>`.
+   */
+  id?: string;
+
+  /**
+   * `true` to use the light version. For use on $ui-01 backgrounds only.
+   * Don't use this to make tile background color the same as the container background color.
+   * @deprecated This prop is no longer needed and has been deprecated in v11 in favor of the new Layer component. It will be moved in the next major release.
+   */
+  light: boolean; // Deprecated: This prop is no longer needed and has been deprecated in v11 in favor of the new Layer component. It will be moved in the next major release.
```

Comment:
```suggestion
   * @deprecated This prop is no longer needed and has been deprecated in v11 in favor of the new Layer component. It will be removed in the next major release.
   */
  light: boolean;
```